src/plugins/intel_gpu/tests/test_cases/concatenation_gpu_test.cpp

```diff
@@ -1285,4 +1285,154 @@ INSTANTIATE_TEST_SUITE_P(smoke,
                             TestParamType_concat(1, { 8, 32 }, 2, 2, false)
                         ),
                         concat_gpu::PrintToStringParamName);
+
+template <typename Type>
+struct concat_gpu_4d_explict : public concat_gpu {
+public:
+    cldnn::memory::ptr run_concat_network(std::vector<std::vector<std::vector<std::vector<std::vector<Type>>>>> input, format::type fmt, ExecutionConfig config) {
+        auto data_type = type_to_data_type<Type>::value;
+        auto& engine = get_test_engine();
+        const size_t batch_num = testing::get<0>(GetParam());
+        const std::vector<size_t> in_features = testing::get<1>(GetParam()); // only use first element.
+        const size_t input_y = testing::get<2>(GetParam());
+        const size_t input_x = testing::get<3>(GetParam());
+        size_t output_f = in_features[0];
+
+        topology topology;
+
+        std::vector<memory::ptr> in_memory;
+        std::vector<primitive_id> input_ids;
+
+        // input0 -- eltwise -- conv -- concat -- output
+        //         /         \        /
+        // input1 -           --------
```

Comment:
what about using `pooling` instead of `conv`? You can remove code to generate weight :)

Reply:
Use conv to select oneDNN impl.